projects/saturn/src/main/java/io/fairspace/saturn/services/views/ViewRowCollection.java

```diff
@@ -0,0 +1,27 @@
+package io.fairspace.saturn.services.views;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+
+public class ViewRowCollection {
+
+    private Hashtable<String, List<ViewRow>> data = new Hashtable<String, List<ViewRow>>();
```

Comment:
Hashtable is a legacy class and not recommended for new code due to its synchronization overhead and other limitations, I would suggest HashMap here